contracts/HubPool.sol

```diff
@@ -73,7 +73,15 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
 
     // Interest rate payment that scales the amount of pending fees per second paid to LPs. 0.0000015e18 will pay out
     // the full amount of fees entitled to LPs in ~ 7.72 days, just over the standard L2 7 day liveness.
-    uint256 lpFeeRatePerSecond = 1500000000000;
+    uint256 public lpFeeRatePerSecond = 1500000000000;
+
+    mapping(address => uint256) public unclaimedAccumulatedProtocolFees;
+
+    // Address that captures protocol fees. Accumulated protocol fees can be claimed by this address.
+    address public protocolFeeCaptureAddress;
+
+    // Percentage of lpFees that are captured by the protocol and claimable by the protocolFeeCaptureAddress.
+    uint256 public protocolFeeCapturePct;
```

Comment:
Same, should change to uint64

Reply:
Can we also put all the uint64 variables next to one another to decrease the cost of reading?